Controllers/PostController.cs

```diff
@@ -300,4 +300,51 @@ public IActionResult Approve(int id)
 
     }
 
+    [HttpGet("{id}/subscribedPosts")]
+    public IActionResult GetSubscribedPosts(int id)
+    {
+        List<Subscription> subs = _dbContext.Subscriptions.Where(s => s.FollowerId == id).ToList();
+
+        var commonAttribute = subs.Select(s => s.CreatorId).ToList();
```

Comment:
Change from var to a list of integers, and change the variable name from `commonAttribute` to something more descriptive